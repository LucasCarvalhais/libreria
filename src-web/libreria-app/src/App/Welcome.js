import React from 'react';

export const Welcome = () => 
    <div>
        <h1>¡Bienvenido a Librería de Madrid!</h1>
        <p>Elige las opciones arriba para navegar en nuestro sitio</p>
    </div>;