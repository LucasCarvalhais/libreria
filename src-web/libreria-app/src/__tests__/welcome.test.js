import React from 'react';
import renderer from 'react-test-renderer';
import Welcome from '../welcome';

test('Welcome has a valid snapshot', () => {
    const component = renderer.create(
        <Welcome />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});