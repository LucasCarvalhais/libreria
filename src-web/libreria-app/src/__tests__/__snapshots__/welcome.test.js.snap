// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome has a valid snapshot 1`] = `
<div>
  <h1>
    ¡Bienvenido a Librería de Madrid!
  </h1>
  <p>
    Elige las opciones arriba para navegar en nuestro sitio
  </p>
</div>
`;
